base/initdefs.jl

```diff
@@ -296,22 +296,39 @@ end
 const atexit_hooks = []
 
 """
-    atexit(f)
+    atexit(f; exitwith = 0)
```

Comment:
Why not call this `exitcode`?

Reply:
I’m fine with that! @JeffreySarnoff preferred `exitwith`, which is why I picked that, but it’s easy enough for me to change it to `exitcode`.

Reply:
if you prefer -- I'm unlikely to use the variable whatever it is named :).

my reasoning: `exitcode` is usually implies every process provides a determinative exit code where that is not our practice (because we write code that works with code that works and handle exceptions exceptionally). The activity here is more an additional, occasional, specific expression.

`exitwith` is more on point as it says "exit, carrying this value" and `exitcode` says ?enter the code to open the door marked exit? (kidding).

Reply:
My reasoning is that "exit code" is the phrase that you use when describing what this is.

Reply:
be my guest -- I feel we have discussed what follows exit as well as any could.

Reply:
do it Dilum

Reply:
Done!